integration

```diff
@@ -953,6 +953,7 @@
   "Master13011/Mailcow-HA",
   "Master13011/vacances-scolaire-HA",
   "Mat931/digitalstrom-homeassistant",
+  "matbott/ha-inumet-uruguay",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>